frontend/package.json

```diff
@@ -31,6 +31,7 @@
     "@cypress/react": "^5.4.0",
     "@material-ui/core": "^4.11.4",
     "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.58",
```

Comment:
Why do we need this experimental, unstable library?

Reply:
Autocomplete is a component of material-ui/lab. Not sure if there's something comparable in material-ui/core (have not find something there). If you know some material-ui alternative let me know!

Reply:
I'm not entirely happy with using this experimantal tech, but leave it for now.